### PR TITLE
sacct, scancel, srun: add page

### LINKED ISSUE #2068: page requests: Slurm commands
Slurm is a job scheduler widely used on supercomputers. There are a few commands used to interface with it that I think it would be useful to add to tldr: https://slurm.schedmd.com/man_index.html, https://slurm.schedmd.com/pdfs/summary.pdf

Commands:

- [x] [`sacct`](https://slurm.schedmd.com/sacct.html) (displays accounting data for all jobs and job steps in the Slurm job accounting log or Slurm database)
- [x] [`sacctmgr`](https://slurm.schedmd.com/sacctmgr.html) (Used to view and modify Slurm account information.)
- [x] [`salloc`](https://slurm.schedmd.com/salloc.html) (Obtain a Slurm job allocation (a set of nodes), execute a command, and then release the allocation when the command is finished.) → #11651
- [x] [`sattach`](https://slurm.schedmd.com/sattach.html) (Attach to a Slurm job step.) → #11651
- [x] [`sbatch`](https://slurm.schedmd.com/sbatch.html) (Submit a batch script to Slurm.)
- [x] [`sbcast`](https://slurm.schedmd.com/sbcast.html) (transmit a file to the nodes allocated to a Slurm job.) → #11651
- [x] [`scancel`](https://slurm.schedmd.com/scancel.html) (Used to signal jobs or job steps that are under the control of Slurm.)
- [x] [`scontrol`](https://slurm.schedmd.com/scontrol.html) (Used view and modify Slurm configuration and state.)
- [x] [`scontab`](https://slurm.schedmd.com/scrontab.html) (Manage Slurm crontab files.) → #11651
- [ ] [`scrun`](https://slurm.schedmd.com/scrun.html) (An OCI runtime proxy for slurm.) → #11991
- [x] [`sdiag`](https://slurm.schedmd.com/sdiag.html) (scheduling diagnostic tool.) → #11651
- [x] [`sinfo`](https://slurm.schedmd.com/sinfo.html) (view information about Slurm nodes and partitions.)
- [x] [`sprio`](https://slurm.schedmd.com/sprio.html) (view the factors that comprise a job's scheduling priority) → #11651
- [x] [`sh5util`](https://slurm.schedmd.com/sh5util.html) (merge utility for acct_gather_profile plugin.) → #11651
- [x] [`squeue`](https://slurm.schedmd.com/squeue.html) (view information about jobs located in the Slurm scheduling queue.)
- [x] [`sreport`](https://slurm.schedmd.com/sreport.html) (Generate reports from the slurm accounting data.)
- [x] [`srun`](https://slurm.schedmd.com/srun.html) (Run parallel jobs)
- [x] [`sshare`](https://slurm.schedmd.com/sshare.html) (Tool for listing the shares of associations to a cluster.) → #11651
- [x] [`sstat`](https://slurm.schedmd.com/sstat.html) (Display various status information of a running job/step.)
- [x] [`strigger`](https://slurm.schedmd.com/strigger.html) (Used set, get or clear Slurm trigger information.) → #11651
- [x] [`sview`](https://slurm.schedmd.com/sview.html) (graphical user interface to view and modif. Slurm state.) → #11651

Daemons/other:

- [x] [`slurmctld`](https://slurm.schedmd.com/slurmctld.html) (The central management daemon of Slurm.) → #11133
- [x] [`slurmd`](https://slurm.schedmd.com/slurmd.html) (The compute node daemon for Slurm.) → #11133
- [x] [`slurmdbd`](https://slurm.schedmd.com/slurmdbd.html) (Slurm Database Daemon.) → #11133
- [x] [`slurmrestd`](https://slurm.schedmd.com/slurmrestd.html) (The Slurm REST API daemon.) → #11133
- [x] [`slurmstepd`](https://slurm.schedmd.com/slurmstepd.html) (The job step manager for Slurm.) → #11133
[~`spank`~](https://slurm.schedmd.com/spank.html) (Slurm Plug-in Architecture for Node and job (K)control) → #11133

### PR DESCRIPTION
**Pull request by cbohn4**
- [x] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

Adding additional slurm commands to help close #2068

### CODE REVIEW

File: pages/linux/sacct.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# sacct
+
+> Display accounting data from the Slurm service.
+> More information: <https://slurm.schedmd.com/sacct.html>.
+
+- Check basic items such as elapsed time, job name, cpus requested, and memory requested:
+
+`sacct -j {{job_id}} --format=elapsed,jobname,reqcpus,reqmem`
```
Comment (navarroaxel) [inline pages/linux/sacct.md:8]: We prefer the long argument format and the argument options should be between `{{}}`:
```suggestion
`sacct --jobs={{job_id}} --format={{elapsed}},{{jobname}},{{reqcpus}},{{reqmem}}`
```

File: pages/linux/sacct.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# sacct
+
+> Display accounting data from the Slurm service.
+> More information: <https://slurm.schedmd.com/sacct.html>.
+
+- Check basic items such as elapsed time, job name, cpus requested, and memory requested:
```
Comment (navarroaxel) [inline pages/linux/sacct.md:6]:
```suggestion
- Display elapsed time, job name, number of requested CPUs, and memory requested of a job:
```

File: pages/linux/sacct.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# sacct
+
+> Display accounting data from the Slurm service.
+> More information: <https://slurm.schedmd.com/sacct.html>.
+
```
Comment (navarroaxel) [inline pages/linux/sacct.md:5]: Could you add the example for just `sacct` without params and `sacct --brief` (looks useful to see what is running)?

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @cbohn4, thanks for your submission! I left some comments.

File: pages/linux/sacct.md (hunk 1 above)
Comment (cbohn4) [reply pages/linux/sacct.md:8]: With `sacct` the `--format` option allows you to specify different fields you want to be displayed, in this case, elapsed, jobname, requested CPUs, and requested memory. I am double checking that the curly brackets around the fields would be correct in this situation, or would it be better to use:
`sacct --jobs={{job_id}} --format={{elapsed,jobname,reqcpus,reqmem}}`

 I have resolved the other comments in the latest commit.

Comment (navarroaxel) [reply pages/linux/sacct.md]: Checking the `tokei` page, is `{{}}` for each field.

**Review by navarroaxel — APPROVED** [review]
LGTM, thanks @cbohn4

**Review by sbrl — APPROVED** [review]
Nice one, @cbohn4!

**Pull request merged**